hr_attendance_reason/models/res_company.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2023 Tecnativa - Víctor Martínez
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    show_in_kiosk_mode = fields.Boolean(string="Show reasons")
```

Comment:
As this is a general model, you need to be explicit in the name:

```suggestion
    attendance_reasons_in_kiosk = fields.Boolean(string="Show attendance reasons in Kiosk")
```